data/human/human missions.txt

```diff
@@ -6890,3 +6890,1716 @@ mission "Hogshead's Dilemma II"
+			choice
+				`	(Open the crate.)`
+					goto crate
+				`	(Dump the crate out the airlock.)`
+
+			action
+				fail
+			`	As the crate tumbles out into lifeless space your onboard sensors quickly note the anomalous heat signature disappear. Whatever was in that crate is no longer your problem, and probably won't be anyone else's either.`
+
+			label crate
+			`	You pry open the crate. It should be full of meals for you and your crew, but after removing the lid you discover a dirty and disheveled man.`
+			`	Before you have a chance to say anything the man looks up at you with narrowed, pained eyes, and whimpers. "P-p-please. I just had to get away. I-I-It wasn't safe. I had to get away." He weakly holds up his hands.`
+			choice
+				`	"Who are you?"`
+				`	"Where's my food?"`
+
+			`	T-T-Timothy Radrickson. I-I tossed your food out to make room. I am sorry. I am so sorry. I have no money to pay you back. I just h-h-had to leave. They were after me. I owned money. I had nothing. Please. Any other Republic world will do. I just need to get away. I just need to start anew. I'll p-p-pay you back I swear. I swear. I will. I swear.`
+			choice
+				`	"Fine. I'll drop you off at the next port."`
+					goto port
+				`	"Is there somewhere you'd like to go?"`
+					goto somewhere
+				`	"I know where you're going. The airlock."`
+
+			action
+				fail
+			`	The feeble man screams as you drag him out of the crate and toward the airlock. He ineptly squirms as you shove him pass the bulkhead seals. He falls to his knees, shudders with terror, and his face leaks tears. You close the airlock, silencing his shrill screams, and hit the button to open it on the outside. The man is promptly sucked out into cold, cruel space and crystallizes before becoming this system's newest comet.`
+
+			label port
+			action
+				fail
+				set "timothy radrickson: port"
+			`	Tears squeeze out the man's sad eyes. "Thank you," he whispers, "thank you."`
+			`	You get the man cleaned up and settled and head back to your station.`
+
+			label somewhere
+			`	Timothy looks at you, his eyes wide and full of tears. "Rand," he whispers. "Rand. I can make good money there. I can turn my life around. B-b-but I'd be okay with anywhere. You d-d-don't have to.."`
+			`	You cut him off. "It's fine, Timothy. I'll take you to Rand."`
+			`	Timothy breaks down in tears. "Thank you. Thank you. Thank you..." he mumbles over and over again.`
+			`	You help him out of the crate and get him cleaned up.`
+				accept
+
+	on complete
+		event "timothy radrickson: on rand" 340
+		conversation
+			`During the trip Timothy's spirits and health improved, and by the time you arrive on Rand he seems like a different man. Before he steps out of the airlock he turns to you one more time. "Thank you so much, Captain <last>." He shakes your hand."`
```

Comment:
```suggestion
			`During the trip Timothy's spirits and health improved, and by the time you arrive on Rand he seems like a different man. Before he steps out of the airlock he turns to you one more time. "Thank you so much, Captain <last>." He gives you a handshake."`
```
Just personal preference.

Reply:
I like the more direct way it's currently written.